collect_app/src/main/java/org/odk/collect/android/tasks/FormLoaderTask.java

```diff
@@ -276,6 +279,35 @@ protected FECWrapper doInBackground(Void... ignored) {
         return data;
     }
 
+    private void logFormDetails(File formFile, File formMediaDir) {
+        long formFileBytesSize = formFile.length();
+
+        List<String> csvFileBytesSizes = new ArrayList<>();
+        File[] files = formMediaDir.listFiles();
+        if (files != null) {
+            for (File mediaFile : files) {
+                if (mediaFile.getName().endsWith(".csv")) {
+                    csvFileBytesSizes.add(mediaFile.length() + "B");
+                }
+            }
+        }
+
+        if (csvFileBytesSizes.isEmpty()) {
+            Timber.i(
```

Comment:
We currently only send warnings and errors to Crashlytics which explains why the log lines we were seeing are warnings. See `CrashReportingTree`.

I think it's ok to use `w` for these even though they're not technically warnings. We don't expect these to stick around for very long.